### ISSUE TITLE
What do you have against Alfheim?

### ISSUE THREAD

**Issue by brandyyn** [AirBurn0/am2playground#3]
I tried to play with your mod today and added it to my pack, confused as to why it stops launching without a crash I check the logs, and it says 

> Alfheim is not supported

Why? What do you have against Alfheim so much that I can't play with the two mods together? 
![image](https://github.com/AirBurn0/am2playground/assets/11356565/edde8960-f9d0-4e2f-a776-0a8d66cce0e2)
![image](https://github.com/AirBurn0/am2playground/assets/11356565/e323d14e-446c-4bc6-ac51-f6f41e8279cf)

**Comment by AirBurn0** [comment]
> 
 ![image](https://user-images.githubusercontent.com/11356565/265682757-edde8960-f9d0-4e2f-a776-0a8d66cce0e2.png) ![image](https://user-images.githubusercontent.com/11356565/265682786-e323d14e-446c-4bc6-ac51-f6f41e8279cf.png)
 
 Thx for reporting readme bug, funny missclick paste.
 
> I tried to play with your mod today and added it to my pack, confused as to why it stops launching without a crash I check the logs, and it says
> 
> > Alfheim is not supported
> 
> Why? What do you have against Alfheim so much that I can't play with the two mods together?

Currently my addon working kinda incorrect with alfheim content, so I just disabled it so noone report errors that I'm already aware of.
In short, I don't recommend my addon while you have Alfheim by now.
Will I ever fix it? Maybe.
Or maybe I'm just add some TiC integration in next addon version, who knows.

But you got the point, I will add this to readme

**Comment by brandyyn** [comment]
what is working incorrectly? surely the bugs arent worth doing all this for... i think this is very unnecessary and it would be better just to add a warning than completely stopping the game from launching ...

**Comment by AirBurn0** [comment]
> what is working incorrectly? surely the bugs arent worth doing all this for... i think this is very unnecessary and it would be better just to add a warning than completely stopping the game from launching ...

If you wish, I can try disabling my addon's content instead of stopping the game

**Comment by AirBurn0** [comment]
> > > what is working incorrectly? surely the bugs arent worth doing all this for... i think this is very unnecessary and it would be better just to add a warning than completely stopping the game from launching ...
> > 
> > 
> > If you wish, I can try disabling my addon's content instead of stopping the game
> 
> what is so broken that you need to do this for? i'm curious.

Massive content spoilers involved so I continue to keep my secrets
![image](https://github.com/AirBurn0/am2playground/assets/69804577/51f08d90-995a-4b9f-bec5-ee96aa8c6354)

**Comment by AirBurn0** [comment]
> > what is working incorrectly? surely the bugs arent worth doing all this for... i think this is very unnecessary and it would be better just to add a warning than completely stopping the game from launching ...
> 
> If you wish, I can try disabling my addon's content instead of stopping the game

I will try to work this around when I'm done with other content
